doc/profile_handling.md

```diff
@@ -0,0 +1,104 @@
+# Profile Handling
+
+This document describes how the profile is handled by AutoYaST during auto-installation/upgrade. If
+you are not interested in the details, having a look at the `Overview` section should be enough.
+
+Additionally, Linuxrc can be involved in the process of getting the AutoYaST profile. You can find
+the details [in the Linuxrc
+repository](https://github.com/openSUSE/linuxrc/blob/ed2699964a9b4305ad37b280a1168901b1f81c78/linuxrc_and_autoyast.md).
+
+## Overview
+
+During the autoinstallation/autoupgrade, an AutoYaST profile goes through different phases:
+
+1. Fetching: the profile is retrieved from the given location and stored in a path, determined by
+   `Yast::AutoinstConfigClass#xml_tmpfile` (usually `/tmp/profile/autoinst.xml`). It is possible to
+   combine multiple files by using the *rules and classes*.
+2. Import: the contents from the profile are imported through the {Yast::ProfileClass#ReadXML Profile#ReadXML}
```

Comment:
I think it should be `{Yast::ProfileClass#ReadXML Profile.ReadXML}`

Reply:
To be honest, I was unsure about that. I will update other cases too.